frontend/src/components/regions/CardDetails/ReviewPopUp.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import Markdown from "react-markdown";
+import Box from "@material-ui/core/Box";
+import Modal from "../../widgets/Modal";
+import Card from "@material-ui/core/Card";
+import { makeStyles } from "@material-ui/core/styles";
+import ReviewStatus from "../../widgets/ReviewStatus";
+import { CardActions, CardContent, Typography } from "@material-ui/core";
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+
+const useStyles = makeStyles((theme) => ({
+  cardStyle: {
+    display: "block",
+    position: "relative",
+    maxWidth: "90vw",
+    maxHeight: "90%",
+  },
+  timeStyle: {
+    fontSize: "11px",
+  },
+  reviewerEmailStyle: {
+    fontSize: "100%",
+    fontWeight: "bold",
+  },
+  CardContent: {
+    fontSize: "11px",
+    padding: "20px",
+    paddingRight: "30px",
+    overflowY: "scroll",

```

Comment:
overflow Y should probably be auto. The scrollbar should only show up when a scrollbar is needed.